Unix/tools/auth_tests_cleanup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+ntlm_file=$HOME/.ntlm
```

Comment:
if this is coming and going with tests then it should go in the output/tmp directory